chef-training/chef-intermediate
# Updates for Chef Fundamentals refresher, #50

* Logo in the footer
  * Slide 1

* Slide 8
  * Update contents of the working directory

* Slide 9 (new)
  * Add an exercise but don't give the answer (yet)

* Slide 12 - delete
  * No longer need to the org validator pem
    * https://www.chef.io/blog/2015/04/16/validatorless-bootstraps/

* Slide 15
  * Delete, it's an empty slide with just a header

* Slide 15
  * Delete.  Slide header was "What have we just done?"

* Slide 15 (new)
  * Add an exercise but don't give the answer (yet)

* Slide 16
  * Upgrade to latest versions of the cookbooks
    * chef-client 3.4.0 -> 4.3.0
    * chef_handler 1.1.9 (new)
    * cron 1.2.8 -> 1.6.1
    * logrotate 1.4.0 -> 1.9.2
    * ntp 1.5.4 -> 1.8.6
    * windows 1.37.0 (new)

* Slide 18
  * webserver role has been renamed to web

* Slide 20
  * set bootstrap-version to 12.3.0
  * include the run list

* Slide 23 - 27
  * no longer passing the validator info on bootstrap
  * fix the bootstrap command:  no space after the "- N", add an ellipses

* Slide 29
  * add ohai directory
  * remove validation.pem
  * Update chef-client version to 12.3.0

* Slide 30
  * Hide slide
  * Renamed the role from 'webserver' to 'web'

* Slide 32
  * hide, we don't have the ssl issue anymore

* Slide 34
  * Add an instructor note about which ports are listening
